Jmiglesias

jmig21 opened the pull request.

Commit:
agrego 1 personaje + 2 items

Commit:
commit

WalterTano reviewed (changes requested):
Es necesario incluir la sentencia "Using Library" al inicio, para poder usar las clases que creamos en el proyecto de librería.
La raza (en el primer personaje que creas sería "Enano") quedó como una clase al final, así que para incluir la raza de un personaje, primero tendrías que crear una instancia de esa raza y pasársela al constructor de Personaje. La creación de la instancia de Raza quedaría más o menos así:
`Raza enano = new Raza("Enano");`

Para los hechizos y la compatibilidad de los items, es necesario siempre enviar una instancia de la clase ArrayList. No es posible enviar "todos" para incluir a todas las razas como compatibles (en todo caso podrías enviar un ArrayList que contenga una instancia de todas las razas que hayamos creado), o "ninguno" para indicar que un item no tiene hechizos (podrías enviar un ArrayList vacío para esto).

Commit:
correcciones de instanciaciones + arrays

Commit:
corrección de deletreo en using Library

Commit:
correcciones varias luego de correr dotnet build

WalterTano committed:
Merge branch 'master' into jmiglesias

WalterTano committed:
Corrección del uso de los constructores de Item

Mala mía acá, realicé la corrección antes de ver que habíamos removido la lista de hechizos del constructor de item.
